src/Schema/Events/Authentication.ts

```diff
@@ -160,3 +161,25 @@ export type AuthTrigger = "click" | "timed"
  * the service the user used to authenticate
  */
 export type AuthService = "apple" | "email" | "facebook"
+
+/**
+ * A user provides us with important data during the onboarding flow
+ *
+ * This schema describes events sent to Segment from [[OnboardingUserInputData]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "onboardingUserInputData",
+ *    context_module: "onboardingInterests",
+ *    context_page_owner_type: "onboarding",
+ *    data_input: "[learnAboutArt]"
+ *  }
+ * ```
+ */
+export interface OnboardingUserInputData {
+  action: ActionType.onboardingUserInputData
+  context_module: ContextModule
+  context_page_owner_type: PageOwnerType
```

Comment:
Do we do onboarding for mobile? We might want to go with the generic `context_owner_type`

Reply:
Yup it's on mobile too, I can update this.